app/navigation/AuthenticatedNavigator.scala

```diff
@@ -103,7 +102,7 @@ class AuthenticatedNavigator @Inject()() extends Navigator {
     if (mode == NormalMode) {
       userAnswers.get(YourEmployerPage) match {
         case Some(true) =>
-          YourAddressController.onPageLoad(mode)
+          HowYouWillGetYourExpensesController.onPageLoad()
         case Some(false) =>
           UpdateEmployerInformationController.onPageLoad(mode)
```

Comment:
This would need logic to only reach this page if current year is in the selection